robots.txt

```diff
@@ -5,32 +5,27 @@
 User-agent: *
 
 # do not search root directory by default.
-Disallow: /
-Allow: /_sitemap/
-
-# but allow some subdirectories:
-Allow: /stable/
-
-# versions should be searched.  Note sitemap.xml downweights them...
-Allow: /5.*/
-Allow: /4.*/
-Allow: /3.*/
-Allow: /2.*/
-Allow: /1.*/
-
-# packages that should still be searched:
-Allow: /cmocean/
-Allow: /mpl-probscale/
-Allow: /mpl_toolkits/
-Allow: /xkcd/
-Allow: /cycler/
-Allow: /matplotblog/
-Allow: /pytest-mpl/
-Allow: /basemap/
-Allow: /mpl-bench/
-Allow: /mpl-altair/
-Allow: /devdocs/
-
+Disallow: /users/
+Disallow: /tutorials/
+Disallow: /thirdpartypackages/
+Disallow: /resources/
+Disallow: /glossary/
+Disallow: /gallery/
```

Comment:
Missed `/gallery.html`?

Reply:
Yeah I wasn't sure about some of the ones that are not in 3.3.3